py-polars/polars/expr/datetime.py

```diff
@@ -875,12 +875,39 @@ def ordinal_day(self) -> Expr:
         return wrap_expr(self._pyexpr.dt_ordinal_day())
 
     def time(self) -> Expr:
+        """
+        Extract time.
+
+        Applies to Datetime columns only; fails on Date.
+
+        Returns
+        -------
+        Time
```

Comment:
This is a little confusing, since an expression is returned, not a python `time` object.

I'd suggest to leave out the Returns section completely, and just be very clear in the single line docstring summary what is happening.

Same for the other 2 methods here.

Reply:
I have aligned with the other docstrings here, see for example a couple methods above:
```
        Returns
        -------
        Date/Datetime series
```
Just to be sure, it returns a `pl.Time` (capital T), should I add `pl.` prefixes instead?

Reply:
I would say "`Expression of data type Time`".

Reply:
Done. Taken the opportunity to remove the references to `Series` in this `Expr` module, this is clearly copy-pasted from the equivalent Series module.